### ISSUE TITLE
Allow to configure the protos root path

### ISSUE THREAD

**Issue by c3s4r** [devsu/condor-framework#41]
When loading proto files, sometimes it's needed to call load using not only the proto filename, but also pass the root of the protos. This is needed for example when a proto file reference imports other packages.

```proto
syntax = "proto3";
package sample;
import "google/api/annotations.proto";
```

Proposal: 

- Allow an optional parameter `rootProtoPath` when calling the `Condor` constructor.
- Allow to pass options to `addService()`. For now the only option would be `rootProtoPath`. This way a user can override the default, if needed.

If the `rootProtoPath` option is passed to any, the `Condor` constructor or the `addService()` method, it must be used to call `grpc.load`like this:

```js
const grpcObject = grpc.load({
  'root': rootProtoPath, 
  'file': serviceDefinition.protoFilePath
});
```

This should be done in `server.js` (around line 35).

**Comment by c3s4r** [comment]
Also, when protoRootPath is set, the call to addService could be different:

```js
server.addService('sample/sample.proto', 'sample.GreeterService', new Greeter());
```

instead of 

```js
server.addService('./protos/sample/sample.proto', 'sample.GreeterService', new Greeter());
```

So, we might need to remove the logic that throws this error: `'Cannot perform operation: ProtoFilePath is not a valid file path'`.

**Comment by c3s4r** [comment]
Implemented in `1.4.0`. `addService()` was deprecated in favor of `add()` which is more friendly to use.
